frontend/src/modules/results/utils.ts

```diff
@@ -139,6 +139,12 @@ const commonFiltersWithoutTrekSelector = [
 export const formatTextFilter = (textFilter: string | null): { q: string } | undefined =>
   textFilter !== null ? { q: textFilter } : undefined;
 
+export const formatDateFilter = (
+  dateFilter: DateFilter | null,
+): { dates_before?: string; dates_after?: string } => {
+  return { dates_before: dateFilter?.endDate, dates_after: dateFilter?.beginDate };
+};
+
```

Comment:
Following the previous review
```suggestion
export const formatDateFilter = (dateFilter: DateFilter | null): DateFilter => {
  return { dates_before: dateFilter.endDate, dates_after: dateFilter.beginDate };
};

```
?

Reply:
@dtrucs If I remove the possibility to dateFilter to be undefined, I've got a servor error if I don't have a beginDate or endDate. That's why i add the "?" in my return. 

![image](https://user-images.githubusercontent.com/99249234/177768879-a9fac5bb-e48f-49b2-acef-cf4b088f030f.png)

That's what i've made : 
`export const formatDateFilter = (
  dateFilter: DateFilter,
): { dates_before?: string; dates_after?: string } => {
  return { dates_before: dateFilter?.endDate, dates_after: dateFilter?.beginDate };
};`

Reply:
Okay i see! 👍 
Right for the optional chaining, 
The output type `DateFiler`  is till accurate, isn't it?

Reply:
@dtrucs Could you clarify your question? I'm not sure I understand

Reply:
Oh wait, theses are not the same props!
My bad, it's ok 👍